### ISSUE TITLE
User Toggles for Feature Flags

### ISSUE THREAD

**Issue by curran** [datavis-tech/vizhub-issue-tracker#517]

**Comment by curran** [comment]
Chrome Dev tools has this:
![image](https://user-images.githubusercontent.com/68416/98539130-cc4d8c00-2259-11eb-9908-12f50b3a3100.png)

**Comment by stushurik** [comment]
@curran what about if this tab (experiments) would be include only for paying users?
Free user would get only stable version. Paying user would be able to use cutting edge version (if they want to).
We can do it the same way as private switch: experiments would be shown to all users, but if free user clicks on experimental feature, site would show popup with upgrade link (painkiller instead of vitamins model)

**Comment by curran** [comment]
Yes! I had this same thought exactly. It's a great idea. Something we can add to the pricing page.
